content/en/serverless/guide/serverless_package_too_large.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Serverless Package Too Large
+kind: documentation
+further_reading:
+- link: '/serverless/installation/nodejs'
+  tag: 'Documentation'
+  text: 'Instrumenting Node.js Applications'
+---
+
+This guide helps you troubleshoot the error "Code uncompressed size is greater than max allowed size of 272629760" when instrumenting your Node.js serverless application using the Datadog serverless plugin. Although there have been very few reports of the error when instrumenting applications written in other languages or deployed using other tools, the ideas described below may still apply.
+
+Essentially this error indicates that your function's _uncompressed_ code size exceeds the 250 MB limit. Both your [function package][1] (the .zip artifact containing your function code and dependencies) and the [Lambda layers][2] configured on your function count towards this limit. Therefore you want to examine both of them to narrow down the root cause. 
+
+## Layers
+
+Typically Datadog adds two Lambda layers for instrumentation -- a language-specific library that instruments the function code and the Extension that aggregates, buffers and forwards the observability data to the Datadog backend.
+
+You can inspect the content and size of the Datadog Lambda layers using AWS CLI command [aws lambda get-layer-version][3]. For example, running the following commands gives you links to download the Lambda layers for _Datadog-Node14-x version 67_ and _Datadog-Extension version 19_ and inspect the uncompressed size (~30 MB combined). The uncompressed size varies by layers and versions, and you should replace the layer name and version number with the ones used by your applications.
```

Comment:
You've hardcoded the layer version numbers here. Nick created a variable you can use instead and reference it: https://github.com/DataDog/documentation/blob/master/layouts/shortcodes/latest-lambda-layer-version.html

Reply:
That's actually intentional, because I want to mention that they are ~30MB to give people a quick sense. I did mention later that people should replace the version and layer name, which I assume (based on existing tickets) people do already know.

Reply:
I thought that may be why, just wanted to leave a comment.